src/values/instruction_value.rs

```diff
@@ -840,14 +846,6 @@ impl<'ctx> InstructionValue<'ctx> {
     }
 }
 
-impl Clone for InstructionValue<'_> {
-    /// Creates a clone of this `InstructionValue`, and returns it.
-    /// The clone will have no parent, and no name.
-    fn clone(&self) -> Self {
-        unsafe { InstructionValue::new(LLVMInstructionClone(self.as_value_ref())) }
```

Comment:
We should note this in the next release as a breaking change..

Reply:
The current behavior seems expected, no? Sort of like cloning a String, you get a totally different allocation?

Reply:
The problem is it also has a Copy impl, which has different behavior to the Clone one.
https://rust-lang.github.io/rust-clippy/master/index.html#/non_canonical_clone_impl

Copy docs also say:
> If a type is Copy then its [Clone](https://doc.rust-lang.org/stable/std/clone/trait.Clone.html) implementation only needs to return *self (see the example above).

https://doc.rust-lang.org/stable/std/marker/trait.Copy.html#whats-the-difference-between-copy-and-clone